packages/cmr-client/src/CMR.ts

```diff
@@ -1,81 +1,42 @@
+// @ts-nocheck
```

Comment:
Why is this set?

Reply:
From https://github.com/nasa/cumulus/pull/3176/files 

We should *definitely* understand why this change was made before pulling it in.   That doesn't seem good at all.

Reply:
This one is easy. TS checking was disabled because there was a param that's effectively optional not marked correctly.

https://github.com/nasa/cumulus/pull/3225/commits/ddd82199687fd7ca6cf433e3f1ea43fef064c6c3